landscape.yml

```
@@ -810,6 +810,12 @@ landscape:
             homepage_url: https://bitergia.com/bitergia-analytics
             logo: bitergia.svg
             crunchbase: https://www.crunchbase.com/organization/bitergia
+          - item:
+            name: CHAOSS Metrics
+            homepage_url: https://chaoss.community
```

How about pointing directly to the released metric definitions?

```suggestion
            homepage_url: https://chaoss.community/metrics
```